src/applications/ivc-champva/10-7959C/chapters/applicantInformation.js

```diff
@@ -114,13 +122,15 @@ export const applicantContactInfoSchema = {
       'We’ll contact this phone number if we need to follow up about this form.',
     ),
     applicantPhone: phoneUI(),
+    applicantEmail: emailUI(),
   },
   schema: {
     type: 'object',
     required: ['applicantPhone'],
     properties: {
       titleSchema,
       applicantPhone: phoneSchema,
+      applicantEmail: emailSchema,
```

Comment:
This keyname looks like it'll collide with the one added [in this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/33501) - we can't have two `applicantEmail` keys.

Reply:
You are correct. And that was my PR so you'd think I would remember...fixed!